docs/code-review-guidelines.md

```diff
@@ -0,0 +1,42 @@
+# Code review guidelines
+
+## Background
+
+We use pull requests (PRs) for all code changes. All PRs have to be
+reviewed and approved by at least one other developer before being merged into
+`master`.
+
+The main objective of code reviews is to improve the quality of code that
+reaches production, so that our codebase is maintainable and correct. We want
+to reduce the number of bugs that go live though there are other secondary
+benefits like knowledge sharing.
+
+## Guidelines
+
+### Preparing PRs
+
+- Create a draft PR if you want early feedback for work in progress.
+- We have a template for the PR descriptions. The _Guidance to
+  review_ section should contain any information that a reviewer might find
+  helpful. In particular consider adding:
+  - A list of particular questions that you would like feedback on, e.g. design
+    choices.
+  - Step-by-step instructions on how to manually test the change, whether
+    running locally or using a review app.
+- Consider extending test application data if it would make the PR easier to test.
```

Comment:
I would probably make this point sound a bit stronger. 

Given we don't copy production data into other environments I think it's really important to update the factories and the GenerateTestApplications worker. 
This will not only make it easier to review the PR but also to product review in QA and staging. 
It may also speed up the development if we can assume that the data generated locally is a good representation of production data. I certainly waisted some time by not checking prod data early and discovering that the factories are not up to date 🙈